SwarmHeader header
int32 taskType
int32 headingMode
int32 colavMode

float64 speed
float64 heading

Area area
bool doImidiate